assignment-1/IPL.java

```diff
@@ -0,0 +1,329 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.util.*;
+
+import static java.util.Map.Entry.comparingByValue;
+import static java.util.stream.Collectors.toMap;
+
+
+public class IPL {
+
+    static String[] teamNames = {"CSK", "MI", "RR", "RCB", "SRH", "PBKS", "DC", "KKR"};
+    static String[] venueNames = {"CHENNAI", "MUMBAI", "JAIPUR", "BENGALURU", "HYDERABAD", "CHANDIGARH", "DELHI", "KOLKATA"};
+    static ArrayList<Team> teams = new ArrayList<>();
+    static ArrayList<Match> matches = new ArrayList<>();
+
+    public static void fortyWicketsClub(String teamName) {
+        for (Team team : teams) {
+            if (team.teamName.equals(teamName)) {
+                for (Player p : team.squad) {
+                    if (p.wickets > 40) {
+                        p.display();
```

Comment:
In this task it is asked for 40 wickets, the code should be written such that it can takes wickets as a parameter, so it can filter based on the input no of wickets

Reply:
Okay, couldn't understand main motive of the question, I will update it soon.